config/install/field.field.media.file.field_media_file.yml

```diff
@@ -9,21 +9,23 @@ dependencies:
       - media
   module:
     - file
+_core:
+  default_config_hash: 52m0CtJVAoE3Qvh2AiciSkV8odAgQeRo4yNDGgUA7dc
 id: media.file.field_media_file
 field_name: field_media_file
 entity_type: media
 bundle: file
-label: File
+label: file
 description: ''
 required: true
 translatable: true
 default_value: {  }
 default_value_callback: ''
 settings:
-  file_directory: media/file
-  file_extensions: 'txt rtf doc docx ppt pptx xls xlsx pdf'
+  file_directory: '[date:custom:Y]-[date:custom:m]'
```

Comment:
This is good as it will align better with Acquia's file hosting system as it does not like 1000s of files in the same dir.